src/tooltip/tooltip.ts

```diff
@@ -25,13 +25,17 @@ import {NgbTooltipConfig} from './tooltip-config';
 @Component({
   selector: 'ngb-tooltip-window',
   changeDetection: ChangeDetectionStrategy.OnPush,
-  host: {'[class]': '"tooltip show tooltip-" + placement', 'role': 'tooltip'},
+  host: {
+    '[class]': '"tooltip show tooltip-" + placement + (customClasses ? " " + customClasses : "")',
```

Comment:
Please use ES6 string templates instead of string concatenation.  It's a lot easier to read.